index.html

```diff
@@ -1606,6 +1606,10 @@ <h3>Algorithm</h3>
               <code>@prefix</code> <a>entry</a>, which MUST be a <a>boolean</a>. Otherwise, an
               <a data-link-for="JsonLdErrorCode">invalid @prefix value</a>
               error has been detected and processing is aborted.</li>
+            <li>If the <a>prefix flag</a> of <var>definition</var>is set to <code>true</code>,
+              and its <a>IRI mapping</a> is a <a>keyword</a>,
+              an <a data-link-for="JsonLdErrorCode">invalid term definition</a>
+              has been detected and processing is aborted.</li>
           </ol>
```

Comment:
Might want to say "has the form of a keyword", so that `@the-next-keyword` would be rejected too. This is related to another discussion from the F2F, for which we don't yet have an open issue: we want to discourage the use of things that may be keywords in the future (should => must).